analyses/org.osate.analysis.model/src/org/osate/analysis/model/Activator.java

```diff
@@ -0,0 +1,22 @@
+package org.osate.analysis.model;
```

Comment:
I don't think this needs to be API -> org.osate.analysis.model.internal?

Reply:
Are you saying the analysis meta model should be internal?  But that would prevent analysis implementations in other plug-ins from being able to use it.  Or are you saying the Activator should be internal?

Reply:
Renamed package `org.osate.analysis.model` to `org.osate.analysis.model.internal`